### ISSUE TITLE
Создать компонент панели для отображения информации

### ISSUE THREAD

**Issue by Savokr** [SobakEdu/EDEN-client-app#3]
Необходимо создать новый A-Frame компонент в рамках mozilla hubs, который будет представлять из себя панель с разного рода информацией. Он должен иметь возможность включения/выключения при получении определённого ивента. Также он должен следовать за пользователем по ходу его движения.

**Comment by Eduard953** [comment]
получается надо в пустой комнате в которую я попадаю через create room сделать компонент в виде панели с информацией которая при например нажатие мышки открывает/закрывает информацию? Как следовать за пользователем я пока что не совсем представляю но уже после того как создам саму панель буду разбираться.

Ещё вопрос, мне код надо вписать в правильные файлы eden-client или через Developer tools в мозилле? Я просто видел такой вариант в туториеле aframe.

**Comment by Savokr** [comment]
Да, в целом примерно так, сегодня можем созвонится и я расскажу пару вещей про то, что нужно менять. Код надо писать в новой ветке, которую надо создать от ветки `hubs-cloud`, туда пушить все изменения и потом сделать пулл реквест. Надо менять сами файлы eden-client-app, да.
